src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,47 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("Set 사이즈 조회")
+    @Test
+    void size() {
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @DisplayName("Set에 값이 있으면 True")
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    void contains_only_true(int input) {
+        assertThat(numbers.contains(input)).isTrue();
+    }
+
+    @DisplayName("Set에 값이 있으면 True, 없으면 False")
+    @ParameterizedTest
+    @CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false"}, delimiter = ':')
+    void contains_true_and_false(String input, String expected) {
```

Comment:
```suggestion
    void contains_true_and_false(int input, boolean expected) {
```

별도로 파싱하지 않고 이렇게도 사용이 가능합니다!
참고하시면 좋을 것 같아요. 👍

Reply:
오!! 감사합니다 꿀팁 !